internal/manifests/operator/manifests.go

```diff
@@ -24,5 +25,14 @@ func BuildAll(featureGates configv1alpha1.FeatureGates, namespace string) ([]cli
 		manifests = append(manifests, prometheusRule)
 	}
 
+	if featureGates.Observability.Datasources.CreateDatasources {
```

Comment:
Can you move/adapt this code to https://github.com/grafana/tempo-operator/blob/a701112a9c1b3557b4c3b3d828c84629b33ca597/internal/manifests/manifests.go#L22?

`internal/manifests/operator` is only for objects for the operator itself, not for objects we create for each `TempoStack` instance.

Reply:
Thanks for the explanation, will move!

Reply:
Moved and will start to work on remaining logic.